templates/themes/app/content/link_embed.html.twig

```diff
@@ -1,11 +1,14 @@
-{{ ng_view_content(
-    content,
-    'payload',
-    {
-        params: {
+{% if location is defined %}
+    {{ ng_view_content(
+        content,
+        'payload',
+        {
+            params: {
             label: label,
             suffix: suffix,
             refererLocationId: refererLocationId|default(null),
         },
-    }
-) }}
+        }
+    ) }}
+{% endif %}
+
```

Comment:
Extra new line here.

Reply:
fixed in a78be97ad3bf3b692bc386fffba6c298e8427b93